Hearthstone Deck Tracker/Utility/Helper.cs

```diff
@@ -54,8 +54,8 @@ public static class Helper
 		public static readonly Dictionary<string, string> LanguageDict = new Dictionary<string, string>
 		{
 			{"English", "enUS"},
-			{"Chinese (China)", "zhCN"},
-			{"Chinese (Taiwan)", "zhTW"},
+			{"简体中文", "zhCN"},
+			{"繁體中文", "zhTW"},
```

Comment:
but many Chinese friends of mine think it is needed
and the options of language in BattleNet client are also showed as many language
![options](https://cloud.githubusercontent.com/assets/587992/18412253/5f1c5c02-77bc-11e6-80fd-4d1f1173c103.png)